tests/test_samples.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from cuqipy_fenics.geometry import FEniCSContinuous, FEniCSMappedGeometry, MaternKLExpansion
+from cuqi.distribution import Gaussian
+from cuqipy_fenics.utilities import compute_stats
+import dolfin as dl
+import numpy as np
+import matplotlib.pyplot as plt
+
+@pytest.mark.parametrize("case", ["G_FEM", "G_KL", "G_map"])
+def test_samples_statistics_par_fun(case):
```

Comment:
I think this kind of test code would be cleaner if pytest fixtures was used to initialize the cases.

Reply:
Agreed! I updated the test, and I like this approach. Will be considering it in future tests. Thanks!